<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ListeReservations</class>
 <widget class="QWidget" name="F_ListeReservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>190</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="GLay2">
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="CBx_LieuxDeReservation"/>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ChBx_LieuxDeRetrait">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Lieu de retrait</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ChBx_LieuxDeReservation">
          <property name="text">
           <string>Lieu de réservation</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="CBx_LieuxDeRetrait"/>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="GLay3">
        <item row="2" column="1" colspan="2">
         <widget class="QComboBox" name="CBx_MallesJeux"/>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ChBx_MallesJeux">
          <property name="text">
           <string>Malles/Jeux</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QComboBox" name="CBx_StatutJeu">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ChBx_StatutJeu">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Statut du jeu</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QCheckBox" name="ChBx_NonConfirme">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Non confirmée</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4">
       <layout class="QGridLayout" name="GLay">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="ChBx_DateReservation">
            <property name="text">
             <string>Date de la réservation </string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="Lb_DateInscription">
            <property name="text">
             <string>à</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="DtE_DateResa_Debut">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDateEdit" name="DtE_DateResa_Fin">
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="ChBx_DateEmpruntPrevue">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Date prévue d'emprunt</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDateEdit" name="DtE_DateEmpruntPrevue_Debut">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="Lb_DateNaissance">
            <property name="text">
             <string>à</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDateEdit" name="DtE_DateEmpruntPrevue_Fin">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="displayFormat">
             <string>dd/MM/yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Bt_RAZ">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Remise à zéro des critères</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/RAZ.png</normaloff>:/RAZ.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Bt_Exporter">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Exporter cette liste</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/CSV.png</normaloff>:/CSV.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Bt_Deselection">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Tous désélectionner</string>
          </property>
          <property name="icon">
           <iconset resource="ImagesIcones.qrc">
            <normaloff>:/Decocher_tout.png</normaloff>:/Decocher_tout.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="Line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="Line" name="Line2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCommandLinkButton" name="Bt_SupprimerListe">
     <property name="text">
      <string>Supprimer les réservations sélectionnées</string>
     </property>
     <property name="icon">
      <iconset resource="ImagesIcones.qrc">
       <normaloff>:/Supprimer.png</normaloff>:/Supprimer.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="Le_SQL"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="Lb_NombreResultat">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Nombre de réservations répondant à vos critères :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="Lb_Resultat">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTreeView" name="Tv_ListeReservations">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
